### ISSUE TITLE
pack inspect-builder <non-existent-tag> does not error

### ISSUE THREAD

**Issue by Zanadar** [buildpack/pack#386]
No error when I try to inspect a non-existant or unreachable builder:
```
± zm |master ✓| → pack inspect-builder cloudfoundry/cnb:0.0.40-bionic
Inspecting builder: cloudfoundry/cnb:0.0.40-bionic

REMOTE:
(not present)

LOCAL:
(not present)

```

**Comment by jromero** [comment]
Was going to mark it as a bug but this seems like a feature request. :)

### Acceptance Criteria
**Given** builder 'non/existent' does not exist
**When** I run `pack inspect-builder non/existent`
**Then** I should see output: `ERROR: Unable to find builder '...' locally or remotely.`
**And** the return exit code should be 1

**Comment by Zanadar** [comment]
@jromero Fair enough 😼 however you want to label it is fine by me!

**Comment by jromero** [comment]
Fixed by #406
